src/index.css

```
@@ -0,0 +1,31 @@
+h1 {
+    color: rgb(3, 148, 35);
+    font-style: oblique;text-shadow:color-mix(in hwb longer hue, color percentage, color percentage);
+    display: block;
+    margin-left: auto;
+    margin-right: auto;
+    width: 40%;
+  }
+body {
+    background-image: url("assets/tetris background.png");
+    background-attachment: fixed;
+    background-repeat: no-repeat;
+    background-size: cover;
+
+  } 
+img {
+    display: block;
+    margin-left: auto;
+    margin-right: auto;
+    width: 40%;
+  }
+button {
+    background-color: #baba00; /* Blue */;
```

I think this comment is out of date
```suggestion
    background-color: #baba00; /* Yellow */;
```